src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -1334,6 +1339,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return result
         End Function
 
+        Private Function IsFixInvocation(node As BoundCall) As Boolean
+            ' Quick test to eliminate most calls to something other that Conversion.Fix
+            If Not "Fix".Equals(node.Method.Name) Then
```

Comment:
Should this comparison be case-insensitive (VB-style)?

Reply:
No. That is the actual name of the method. This is not a name lookup.